src/commands/serve.rs

```diff
@@ -17,6 +17,12 @@ pub struct Serve {
 }
 
 impl Serve {
+    pub fn new(port: Option<u16>) -> Self {
+        Serve {
+            port: port.unwrap_or(7979),
```

Comment:
```suggestion
            port: port.or_else(Dev::default().port)
```

if this works, this is still a little noisy to me compared to 7979 but might be nice? wdyt?

Reply:
if this worked i would 100% prefer it heh, let's see